js/events/index.d.ts

```diff
@@ -6,6 +10,50 @@ export interface EventType { }
  */
 export type TEvent = {} extends EventType ? dxEvent : EventType;
 
+/** @public */
+export interface ComponentEvent<T> {
+    component: T;
+    element: TElement;
+    model?: any;
+}
+
+/** @public */
+export interface ComponentNativeEvent<T> {
+    component: T;
+    element: TElement;
+    model?: any;
+    event?: TEvent;
```

Comment:
About this optional modifier - is it correct? The `event` field can be `undefined` even if an event is a *native-like* one?

/cc @DokaRus, @IgnatovDan

Reply:
There is a code for the "args.event is null or undefined" config:
https://github.com/DevExpress/DevExtreme/blob/21_1/js/ui/context_menu/ui.context_menu.js#L861
https://github.com/DevExpress/DevExtreme/blob/21_1/js/ui/pivot_grid/ui.pivot_grid.field_chooser.js#L333

And, we have not checked the 'args.event' value on our tests.

Reply:
Ok, let's keep it optional for now